companion/lib/Internal/Util.ts

```
@@ -47,262 +44,138 @@ export function ParseInternalControlReference(
 		}
 	}
 
-	const parseLocationString = (str: string | undefined): ControlLocation | null => {
-		if (!str) return null
+	const parts = str.split('/') // TODO - more chars
 
-		const parts = str.split('/') // TODO - more chars
+	// nocommit - this is horrible, and needs reworking to be simpler
```

ESLint

[error] Unexpected 'nocommit' comment: 'nocommit - this is horrible, and needs...'.

(no-warning-comments)